### ISSUE TITLE
Export Schema/structure and data

### ISSUE THREAD

**Issue by devpkr1** [dbcodeio/public#94]
Hi Mike,

I have a few suggestions for the context menus of both Database and Table.

For the Database context menu, adding options like **Dump Structure**, **Dump Structure with Data**, and **Import SQL** would be useful. These options would facilitate exporting the database structure and data to an SQL file and importing database structure and data from an SQL file. (Screenshot attached)

![Image](https://github.com/user-attachments/assets/0d52f4dd-7050-4387-92cd-de8bd4f886d8)

For the Table context menu, I suggest adding options such as **Dump Structure**, **Dump Structure with Data**, **Import from SQL and CSV**, and **Generate Mock/Test Data**. These additions would streamline exporting, importing, and generating test data directly within the table context.

Thanks
Pradeep

**Comment by mikeburgh** [comment]
Yes, something I plan to add, it's a bit complex as I want to use the database native tools to do it so the export is 100% accurate and things are not missed when scripting tables etc

**Comment by mikeburgh** [comment]
Going to track this in #195
